packages/terra-date-input/src/DateInput.jsx

```diff
@@ -264,6 +265,26 @@ class DateInput extends React.Component {
     }
   }
 
+  /**
+   * Checks Paste event in the day and year input, and processes it based on the value of the keycode
+   * Prevents non-numeric characters from being entered in Safari browser.
+   * @param {Object} event Event object generated from the event delegation.
+   */
+  handlePaste = event => {
+    const input = (event.clipboardData || window.clipboardData).getData('text');
+    if (!input.match(/^[0-9]+$/)) event.preventDefault();
```

Comment:
Would it work if we validate the key using `DateInputUtil.validNumericInput(input)` so that we are consistent with the other places in how we validate key inputs? Same for `handleKeyPress`.

Something like `if (!DateInputUtil.validNumericInput(input)) event.preventDefault();`

Reply:
Done 78e90ec